### ISSUE TITLE
After install, http://localhost:4321/_tailwind results in 404.

### ISSUE THREAD

**Issue by jjaimealeman** [florian-lefebvre/astro-tailwind-config-viewer#4]
Installed via pnpm on a fairly fresh install.
`pnpm astro add astro-tailwind-config-viewer`

But resulted in 404 error.

![image](https://github.com/florian-lefebvre/astro-tailwind-config-viewer/assets/1428292/0ae2bfd1-c208-4c82-92cf-0ec08c03dc3c)

![image](https://github.com/florian-lefebvre/astro-tailwind-config-viewer/assets/1428292/eb603fae-c707-4da4-915b-13641aa37878)

```
/astro.config.mjs
import { defineConfig } from 'astro/config';
import tailwind from "@astrojs/tailwind";
import icon from "astro-icon";

import tailwindConfigViewer from "astro-tailwind-config-viewer";

// https://astro.build/config
export default defineConfig({
  integrations: [tailwind(), icon(), tailwindConfigViewer()]
});
```

```
/package.json
{
  "name": "website_v3",
  "type": "module",
  "version": "0.0.1",
  "scripts": {
    "dev": "astro dev",
    "start": "astro dev",
    "build": "astro check && astro build",
    "preview": "astro preview",
    "astro": "astro",
    "lint": "prettier --write \"**/*.{js,jsx,ts,tsx,md,mdx,astro}\""
  },
  "dependencies": {
    "@astrojs/check": "^0.6.0",
    "@astrojs/tailwind": "^5.1.0",
    "astro": "^4.8.3",
    "astro-icon": "^1.1.0",
    "astro-tailwind-config-viewer": "^2.0.0",
    "sharp": "^0.33.3",
    "tailwindcss": "^3.4.3",
    "typescript": "^5.4.5"
  },
  "devDependencies": {
    "@types/node": "^20.12.12",
    "eslint": "9.2.0",
    "eslint-plugin-astro": "^1.2.0",
    "prettier": "^3.2.5",
    "prettier-config-standard": "^7.0.0",
    "prettier-plugin-astro": "^0.13.0",
    "prettier-plugin-tailwindcss": "^0.5.14"
  }
}

```

**Comment by florian-lefebvre** [comment]
Can you create a minimal reproduction using https://astro.new/latest (stackblitz)? That would help me a lot

**Comment by jjaimealeman** [comment]
@florian-lefebvre 

Can you believe this is my first time ever using stackblitz? 😄 

So I started a new container and ran `npx astro add astro-tailwind-config-viewer`, accepted all defaults.

![image](https://github.com/florian-lefebvre/astro-tailwind-config-viewer/assets/1428292/9e5c8a44-3413-416b-8d76-281774fb3a63)

The file `tailwind.config.msj` did not exist so I just pasted in my own and restarted the dev server.

![image](https://github.com/florian-lefebvre/astro-tailwind-config-viewer/assets/1428292/688c582b-c167-4bc6-bad6-e291edf8077e)

Same error amigo.

Not sure how to use stackblitz, but here is the URL 
https://stackblitz.com/edit/github-mzwbpb?file=src%2Fpages%2Findex.astro

**Comment by florian-lefebvre** [comment]
perfect thank you!

**Comment by florian-lefebvre** [comment]
Released in 2.0.1!

**Comment by jjaimealeman** [comment]
BRILLIANTLY DONE!!!

`http://localhost:4321/_tailwind/favicon.ico` still shows the Vue favicon, I love it!

Even shows me new color palette!

![image](https://github.com/florian-lefebvre/astro-tailwind-config-viewer/assets/1428292/6b585624-d7a5-4af0-9a0a-7db63c864407)

---

Issue closed my friend 👍🏽
